example-app/src/main/res/values/styles.xml

```diff
@@ -16,7 +16,7 @@
         <item name="colorControlNormal">@color/color_control_normal</item>
         <item name="colorControlHighlight">@color/color_control_highlight</item>
 
-        <item name="android:textColor">?attr/colorOnBackground</item>
+        <item name="android:textColor">@color/text_color_primary</item>
         <item name="android:textColorPrimary">?attr/colorOnBackground</item>
```

Comment:
To make sure the correct text color is used everywhere we should also override `textColorPrimary`, `textColorSecondary` and `textColorTertiary` with `@color/text_color_primary`